tests/test_schema.py

```diff
@@ -342,6 +342,38 @@
     }
   }
 }"""
+SH_INDIVIDUALS_BOT_FILTER = """{
```

Comment:
We should reduce the data that is requested in this query to make the test easier to read and understand it. We only need `mk` and `isBot` fields.